eth/beacon/state_machines/forks/serenity/__init__.py

```diff
@@ -0,0 +1,23 @@
+from typing import Type  # noqa: F401
+
+from eth.beacon.types.blocks import BaseBeaconBlock  # noqa: F401
+from eth.beacon.types.states import BeaconState  # noqa: F401
+
+from eth.beacon.state_machines.base import BeaconStateMachine
+from eth.beacon.state_machines.state_transitions import BaseStateTransition  # noqa: F401
+
+from .blocks import SerenityBeaconBlock
+from .states import SerenityBeaconState
+from .state_transitions import SerenityStateTransition
+from .configs import SERENITY_CONFIG
+
+
+class SerenityStateMachine(BeaconStateMachine):
+    # fork name
+    fork = 'serenity'  # type: str
```

Comment:
@cburgdorf am I correct that we should start using [`ClassVar`](https://docs.python.org/3.5/library/typing.html#typing.ClassVar) for things like this?